types/bittorrent-protocol/index.d.ts

```diff
@@ -56,6 +56,8 @@ declare namespace BittorrentProtocol {
 
         use(ext: ExtensionConstructor): void;
 
+        [key: string]: any;
```

Comment:
Would it be more precise if we  avoid the `any` and use `Extension` instead?

Reply:
I wish it would work like that. But that's not what would happen. All properties would need to be extensions then. And that's obviously not what we want. If you try it, it errors out because all the other previously defined properties aren't extensions.

Reply:
True, in that case keep the `any`